website/components/code/code-block.vue

```diff
@@ -95,18 +73,15 @@ const CodeBlock = props => ({
       ])
     } else {
       const liveEditor = h(LiveEditor, {
-        props: {
-          code
-        }
+        props: { code }
       })
 
-      if (props.browser) {
+      if (process.browser) {
```

Comment:
👍 I don't know how I forgot this 😂

Reply:
This is one of those *"I've been coding the entire night and words are starting to blend so I should close the computer"* mistakes lol

Reply:
Haha for real. 🤣 Was near my mental limit when writing this